lib/testmap.py

```diff
@@ -247,7 +242,7 @@ def contexts(image, *scenarios: Iterable[str]):
         "rhel-8-8@candlepin/subscription-manager/subscription-manager-1.28",
         "rhel-9-3@candlepin/subscription-manager-cockpit",
     ],
-    # Anaconda builds in fedora-37 and runs tests in fedora-rawhide-boot
+    # Anaconda builds in fedora-39 and runs tests in fedora-rawhide-boot
```

Comment:
I believe this was outdated comment.

Reply:
Indeed it was. It should actually be `fedora-rawhide`, as the line directly below says. As this PR is cheap to update, would you mind fixing it?

Reply:
I pushed this myself.